docs/project/library-servicing.md

```diff
@@ -2,21 +2,19 @@
 
 This document provides the steps necessary after modifying a library in a servicing branch (where "servicing branch" refers to any branch whose name begins with `release/`).
 
-## Check for existence of a .pkgproj
+## Check if a package is generated
 
-Your first step is to determine whether or not your library has its own package. To do this, go into the root folder for the library you've made changes to. If there is a `pkg` folder there (which should have a `.pkgproj` file inside of it), your library does have its own package. If there is no `pkg` folder there, the library is the part of the shared framework. If it is, then there is nothing that needs to be done.
-
-For example, if you made changes to [Microsoft.Win32.Primitives](https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Win32.Primitives), then you have a `.pkgproj`, and will have to follow the steps in this document. However, if you made changes to [System.Collections](https://github.com/dotnet/runtime/tree/main/src/libraries/System.Collections), then you don't have a `.pkgproj`, and you do not need to do any further work for servicing.
+If a library's source project sets `<IsPackable>true</IsPackable>` a package is generated. If the library's source project doesn't set `<IsPackable>true</IsPackable>`, the library is the part of the shared framework. If it is, then there is nothing that needs to be done.
```

Comment:
```suggestion
If a library's source project sets `<IsPackable>true</IsPackable>` a package is generated. If the library's source project doesn't set `<IsPackable>true</IsPackable>`, the library is part of the shared framework. If it is, then there is nothing that needs to be done.
```

This is not entirely true, right? In order to be part of the shared framework it has to be listed in `NetcoreAppLibrary.props`?

Reply:
Right this wasn't and isn't true. Let me refine that paragraph.